plugin/src/test/java/io/jenkins/plugins/casc/SchemaGenerationTest.java

```diff
@@ -54,9 +63,32 @@ public void schemaShouldSucceed() {
                         "      \"$ref\": \"#/definitions/" + a.type.getName() + "\"\n" +
                         "    },");
 
+
             }
         }
 
         System.out.println(schemaString);
+
+        /**
+         * Used to generate the schema for the descriptors
+         */
+        ConfigurationAsCode configurationAsCode = ConfigurationAsCode.get();
+
+        configurationAsCode.getConfigurators()
+                .stream()
+                .forEach(configurator -> {
+                    if(configurator.getClass().getSimpleName().equals("HeteroDescribableConfigurator")) {
+                        HeteroDescribableConfigurator b = (HeteroDescribableConfigurator) configurator;
+                        DescriptorExtensionList descriptorExtensionList = j.jenkins.getDescriptorList(b.getTarget());
+
+                        for (Object obj: descriptorExtensionList) {
+                            /*
+                            * Iterate over the obj list and construct the schema.
```

Comment:
This iteration will be used to get the schema built in a string builder for every Descriptor list.